lib/Models/LeafletAttribution.ts

```diff
@@ -0,0 +1,96 @@
+import { IObservableArray, runInAction, observable } from "mobx";
+import L from "leaflet";
+import Terria from "./Terria";
+
+export class LeafletAttribution extends L.Control.Attribution {
```

Comment:
For cesium this is just a modification of the approach that was used for a long time in Terria. I am not sure that it can be done for cesium since most of the API is private, we just hide it and provide our implementation with the minimum usage of private API. Changing to a leaflet approach will require looking into private API and copying it which will make it harder to maintain in future.

Reply:
OK sounds like you have taken the best approach, thanks.